linux/LinuxProcessList.c

```diff
@@ -1172,7 +1172,7 @@ static inline void LinuxProcessList_scanZfsArcstats(LinuxProcessList* lpl) {
    char buffer[128];
    while (fgets(buffer, 128, file)) {
       #define tryRead(label, variable) do { if (String_startsWith(buffer, label) && sscanf(buffer + strlen(label), " %*2u %32llu", variable)) { break; } } while(0)
-      #define tryReadFlag(label, variable, flag) do { if (String_startsWith(buffer, label) && sscanf(buffer + strlen(label), " %*2u %32llu", variable)) { flag = 1; break; } else { flag = 0; } } while(0)
+      #define tryReadFlag(label, variable, flag) do { if (String_startsWith(buffer, label) && sscanf(buffer + strlen(label), " %*2u %32llu", variable)) { (flag) = 1; break; } else { (flag) = 0; } } while(0)
```

Comment:
```suggestion
      #define tryReadFlag(label, variable, flag) do { if (String_startsWith((buffer), (label)) && sscanf((buffer) + strlen((label)), " %*2u %32llu", (variable))) { (flag) = 1; break; } else { (flag) = 0; } } while(0)
```

Reply:
`buffer` is not a marco argument (it comes from the callers scope) and i think it is unnecessary to enclose single function arguments

Reply:
k, for `buffer`. But `label` and `variable` are macro arguments and thus should be enclosed.